roles/bootstrap-os/tasks/main.yml

```diff
@@ -92,6 +92,14 @@
   tags:
     - facts
 
+- name: Set os_family fact for UOS Linux
+  set_fact:
+    ansible_os_family: "RedHat"
+    ansible_distribution_major_version: "8"
+  when: ansible_distribution == "UnionTech"
+  tags:
+    - facts
+
```

Comment:
UOS is supported by the ansible 2.13.
From: https://github.com/ansible/ansible/pull/77275.

So if we upgrade to ansible 2.13 we do not need to add this.

BTW the Uos distribution may be the debian

Reply:
That ansible PR https://github.com/ansible/ansible/pull/77275 is for UOS/deepin desktop user.  This pr is for uos server distro(UnionTech OS Server 20), it is similar to RedHat/CentOS. :)

Reply:
Got it. Thanks 

BTW we can also provide a PR to the ansible to support the 'UnionTech OS' , so that the kubespray can remove this after the ansible upgraded.

Reply:
Yes, thank your advise.